app/views/buses/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Buses</h1>
+
+<h2>by postcode</h2>
```

Comment:
Do you mind adding screenshots for both views and errors to the PR description? Just curious what it looks like